spec/plot_list_spec.rb

```diff
@@ -0,0 +1,79 @@
+describe Daru::View::PlotList do
+  let(:df) do
+    Daru::DataFrame.new(
+      {
+        a: [1, 2, 3, 4, 5, 6],
+        b: [1, 5, 2, 5, 1, 0],
+        c: [1, 6, 7, 2, 6, 0]
+      }, index: 'a'..'f'
+    )
+  end
+  let(:dv) { Daru::Vector.new [:a, :a, :a, :b, :b, :c], type: :category }
+  let(:array1) { [
+      ['Galaxy', 'Distance', 'Brightness'],
+      ['Canis Major Dwarf', 8000, 230.3],
+      ['Sagittarius Dwarf', 24000, 4000.5],
+      ['Ursa Major II Dwarf', 30000, 1412.3],
+      ['Lg. Magellanic Cloud', 50000, 120.9],
+      ['Bootes I', 60000, 1223.1]
+    ] }
+  let(:array2) { [[1, 15], [2, 30], [4, 40]] }
+  let(:options_gc) {{width: 800, height: 720, adapter: :googlecharts}}
+  let(:plot_gc) { Daru::View::Plot.new(array1, type: :bar, adapter: :googlecharts) }
+  let(:table_gc) { Daru::View::Table.new(array1, options_gc) }
+  let(:plot_hc) { Daru::View::Plot.new(
+    array2, chart: { type: 'line' }, adapter: :highcharts
+  ) }
+  let(:plot_nyaplot) { Daru::View::Plot.new(dv, type: :bar, adapter: :nyaplot) }
+  let(:plots_array) { [plot_gc, table_gc, plot_hc, plot_nyaplot] }
+  let(:combined) { Daru::View::PlotList.new(plots_array) }
+  context 'initialize PlotList' do
+    it 'sets correct data' do
+      expect(combined).to be_a Daru::View::PlotList
+      expect(combined.data).to eq plots_array
+    end
+  end
+  describe '#div' do
+    subject (:js) { combined.div }
+    it 'generates a table in js' do
+      expect(js).to match(/<table class='columns'>/)
+      expect(js).to match(/<td><div id=/)
```

Comment:
check 4 times `<div id=` . It would be better if we can test the custom id, that was passed as well.